tests/scripts/arp_responder.py

```diff
@@ -5,12 +5,17 @@
 from collections import defaultdict
 import logging
 import scapy.all as scapy
+import ipaddress
+scapy.MTU = 1280
+scapy.conf.use_pcap = True
+# Callers: Expect to wait 10-15 seconds here, as scapy reinitializes everything to use libpcap
```

Comment:
Can this be avoided somehow by making this as a default behavior so that reinit is not needed?

Reply:
It appears that it's not strictly the fact that scapy is reinitializing everything, but rather that its setup implementation for using libpcap is slow.

Reply:
Based on a quick test on the py3only PTF image (which is based on Bullseye instead of Buster), this might not be an issue there.